requirements.txt

```diff
@@ -23,6 +23,7 @@ Flask-Babel
 money
 pymarc
 accept-types
+watchtower # for cloudwatch logging
```

Comment:
Do the requirements come in recursively, or does this need to be added here?

Reply:
It needs to be added separately. The core requirements are only used when running the unit tests.